spec/models/health_care_application_spec.rb

```diff
@@ -4,6 +4,12 @@
 
 RSpec.describe HealthCareApplication, type: :model do
   let(:health_care_application) { create(:health_care_application) }
+  let(:health_care_application_short_form) do
```

Comment:
I DRY'd up where we were testing the short form in two places to this shared variable